src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1489,12 +1489,16 @@ void runtime::cpu::CPU_ExternalFunction::build()
                 {
                     for (size_t j = 0; j < p.second; j++)
                     {
-                        if (runtime::cpu::IsTracingEnabled())
+                        // we will only update the profile_counter for the first functor within
```

Comment:
This is incorrect (thought I documented it but guess not). There needs to be _exactly_ one functor per op. If you have multiple computation stages then you'll need to build one functor with those stages. The matmul + bias builder has an example. The maxpoolbackprop builder should be fixed.